src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/CommandMode.java

```diff
@@ -246,7 +246,25 @@ public void setBranches(List<BeginAnalysis> branches) {
         public ConfigView(String appName, String branchName,
                           CommandBuildName buildName, String labId, String buildSessionId, String additionalArguments,
                           List<TechnologyOptions> techOptions) {
-            super(CommandModes.Config, buildSessionId, additionalArguments);
+                          this(appName, branchName, buildName, labId, buildSessionId, additionalArguments,
+                           techOptions, CommandModes.Config);
+        }
+
```

Comment:
Not directly related to this code, just saw in Github: we have security issues with our plugin - one of our dependencies is in risk.
Can you please check if that's easy to upgrade?
https://github.com/jenkinsci/sealights-plugin/network/alert/pom.xml/com.fasterxml.jackson.core:jackson-databind/open

Reply:
File not found

Reply:
Sent you in slack.